apps/ui/src/components/FormAuctionBid.vue

```diff
@@ -206,28 +214,6 @@ function handlePlaceOrder() {
   });
 }
 
-function handlePriceUpdate(value: string, fromSlider = false) {
```

Comment:
Moved this function down as `getColor` added in previous PR split two related functions (`handlePriceUpdate` and `handleFdvUpdate`).